labgrid/remote/client.py

```diff
@@ -809,6 +823,21 @@ def create():
     loop.run_until_complete(ready.wait())
     return session[0]
 
+def find_role_by_place(config, place):
+    for role, role_config in config.items():
```

Comment:
If we don't have to pass the role, this can be simplified as well.

Reply:
The role name is part of the return value here as well.